### ISSUE TITLE
Compatible dark mode (breeze dark)

### ISSUE THREAD

**Issue by hervyqa** [insilmaril/vym#53]
Thank you for creating the best mindmap software. I found some bugs for dark mode in Linux distributions generally using Breeze Dark (default KDE dark theme). It's just that in the textbox it can't be changed to the native dark theme form. I also encountered text that cannot be changed to white.

**Tree editor**

![Screenshot_20230126_170009](https://user-images.githubusercontent.com/45872139/214810775-e109f14c-08ab-4678-8396-cbe2da35de0d.png)

**Note editor**

![Screenshot_20230126_170220](https://user-images.githubusercontent.com/45872139/214811033-fbc69e1f-d333-4964-94aa-d254d790f50c.png)

![Screenshot_20230126_170202](https://user-images.githubusercontent.com/45872139/214810943-95ce14ce-be01-4fc9-a7a6-2ed968d3d31d.png)

**Text editor**

![Screenshot_20230126_170313](https://user-images.githubusercontent.com/45872139/214811129-d9eb3a79-7cdd-4c80-83a4-e2bc86963c99.png)

**Script editor**

![Screenshot_20230126_170533](https://user-images.githubusercontent.com/45872139/214811195-cb841e63-d63f-4d82-a24e-652cd71495eb.png)

**Property editor > Task**

![Screenshot_20230126_170704](https://user-images.githubusercontent.com/45872139/214811247-ecf54673-ad70-4a31-9456-deaf3fb9aa19.png)

**Overwriting file**

![Screenshot_20230126_175242](https://user-images.githubusercontent.com/45872139/214818599-485b3198-16ee-4ec1-8b05-c604f2f6558c.png)

**VYM Info**

![Screenshot_20230126_200758](https://user-images.githubusercontent.com/45872139/214843431-35c1985f-bc0d-4a5b-90d5-84e92b90f050.png)

```
vym version: 2.8.42 - 2022-09-29
   Platform: NixOS 22.11 (Raccoon)
 localeName: en_US.UTF-8
  tmpVymDir: /tmp/vym-L0VX6g
zipToolPath: /nix/store/kw19wgs88ka17id6l2cwpyvs5jnphjkn-zip-3.0/bin/zip
 vymBaseDir: /nix/store/m9nwjb77gfvnv9wcwp6hf0yxvfl1aiw7-vym-2.8.42/share
currentPath: /home/hervyqa
 appDirPath: /nix/store/m9nwjb77gfvnv9wcwp6hf0yxvfl1aiw7-vym-2.8.42/bin
   Settings: /home/hervyqa/.config/InSilmaril/vym.conf
 SSL status: supported
     SSL Qt: OpenSSL 3.0.7 1 Nov 2022
    SSL lib: OpenSSL 3.0.7 1 Nov 2022
```

**Comment by insilmaril** [comment]
Thanks - I wasn't even aware that Qt applications could follow dark mode settings from KDE, only noticed dark mode on Mac so far. I am going to look into this, but might take quite some time (currently doing major layout changes. And probably releasing 2.9.0 today ;-)

**Comment by insilmaril** [comment]
Reproduced with KDE Breeze dark theme.

vym 3.x will have more map designs, both for various layouts and color schemes

**Comment by insilmaril** [comment]
Started to work a bit on this:  (9510def33f23fadf3f3725350e9ba96980eb2c12)

- [x]  TreeEditor uses map background color and frame colors Note: selected item still has wrong foreground color :-(
- [x] Property Editor
- [x] Warning Dialog when overwriting file
- [x] vym info
- [ ] NoteEditor/HeadingEditor
- [ ] Scripteditor

The remaining editors can partly be workarounded: For NoteEditor and plaintext the colors can be set in settings. The rest is tricky and will need more time.

**Comment by hervyqa** [comment]
Wooww. Thank you for your hard work!

**Comment by insilmaril** [comment]
- [ ]  NoteEditor/HeadingEditor
- [x] ScriptEditor
- [x] TreeEditor

**Comment by insilmaril** [comment]
I just released 2.9.0 Beta 4 in https://sourceforge.net/projects/vym/files/Development/ 

Dark theme is enabled there. :-)

If I still overlooked something, please open bug tickets.

(Some icons are not optimal yet, going to enhance them later)
